packages/components/tables/_tables.scss

```diff
@@ -40,24 +61,83 @@
 ========================================================================== */
 
 /**
- * Responsive table is used to ensure the table does not break the container.
- *
- * 1. Margin is removed so there isn't double spacing.
+ * 1. Hiding the thead on mobile
+ * 2. Displaying the thead on desktop
+ * 3. Removing default screen reader behaviour
+ * 4. Assigning role of table-row on desktop to give default screen reader behaviour
+ * 5. Using justify content to space out elements in the row on mobile
+ * 6. Assigning a minimum width incase of black cell
+ * 7. Aligning content to the right on mobile
+ * 8. Aligning mobile header to left to split it from the data
+ * 9. Hiding mobile specific header from desktop view
  */
 
 .nhsuk-table-responsive {
-  @include nhsuk-responsive-margin(7, 'bottom');
-
-  display: block; // sass-lint:disable-line property-sort-order
-  -webkit-overflow-scrolling: touch; // sass-lint:disable-line property-sort-order
-  -ms-overflow-style: -ms-autohiding-scrollbar; // sass-lint:disable-line property-sort-order
-  overflow-x: auto; // sass-lint:disable-line property-sort-order
+  margin-bottom: 0;
   width: 100%;
 
-  .nhsuk-table {
-    margin: 0; /* [1] */
+  thead {
+    @include visually-hidden; /* [1] */
+
+    @include mq($from: desktop) {
+      @include visually-shown(table-header-group); /* [2] */
+    }
   }
 
+  .nhsuk-table__body {
+    .nhsuk-table-responsive__heading {
+      font-weight: 700;
+      padding-right: 1em;
+      text-align: left; /* [8] */
+
+      @include mq($from: desktop) {
+        display: none; /* [9] */
+      }
+    }
+
+    .nhsuk-table__row {
+      display: block; /* [3] */
+      margin-bottom: 1.5em;
```

Comment:
Same as above - `margin-bottom` is using `em` but we don't (I think) use `em` any where else. Thinking this should using the equivalent spacing value eg: `nhsuk-spacing(5)`